src/macros/@MPSGEvariable.jl

```diff
@@ -466,6 +498,11 @@ M = MPSGEModel()
 
 @auxiliary(M, X[region=R, goods=G], description="Auxiliary with indexed variables")
 ```
+
+!!! note
+    By default auxiliary variables start at 0 with on lower or upper bounds. These
+    can be set after variable creating using [`set_start_value`](@ref), 
```

Comment:
I'm confused by "0 with on lower or upper bounds".
My undertanding: By default, there's a 0 lower bound. And 0 is (also) the default start value. 
I don't see how the upper bounds is relevant here. And is "on" a typo?

Reply:
"on" should be "no". 

You can see this in the `add_variable!(m::MPSGEModel, S::Auxiliary)` function in `build.jl`. Only the start value of 0 is set. This matches the behavior of MPSGE in GAMS, any bounds on auxiliary variables are set after creation.

This is only true for auxiliary variables, all other (non-parameter) variables get a lower bound of 0.